### PR TITLE
haxe.unit documentation

### PR DESCRIPTION
**Pull request by markknol**

### CODE REVIEW

File: std/haxe/unit/TestCase.hx [hunk 1]
```diff
@@ -22,24 +22,62 @@
...
 	}
 
+	/**
+		Override this method to execute code before the test runs.
+	**/
 	public function setup() : Void {
 	}
 
+	/**
+		Override this method to execute code after the test runned.
```
Comment (Gama11) [inline std/haxe/unit/TestCase.hx]: ran

File: std/haxe/unit/TestCase.hx [hunk 2]
```diff
@@ -22,24 +22,62 @@
...
+	These test methods should call the assertion methods:
+
+	 * `assertTrue(a)`: Succeeds if `a` is `true`.
+	 * `assertFalse(a)`: Succeeds if `a` is `false`.
+	 * `assertEquals(expected, actual)`: Succeeds if `expected` and `actual`
+	   are equal.
+
+	```haxe
+	class MyTestCase extends haxe.unit.TestCase {
+		public function testBasic() {
```
Comment (nadako) [inline std/haxe/unit/TestCase.hx]: I don't think test functions needs to be public

File: std/haxe/unit/TestRunner.hx [hunk 3]
```diff
@@ -22,14 +22,39 @@
...
 #end
 
+	/**
+		Prints the given object/value.
+		
+		 * Flash outputs the result in a new `TextField` on stage.
+		 * JavaScript outputs the result using `console.log`.
+		 * Other targets use native `print` to output the result.
+	**/
 	public static dynamic function print( v : Dynamic ) untyped {
```
Comment (nadako) [inline std/haxe/unit/TestRunner.hx]: I think it's worth noting that this function is `dynamic` so it can be overriden in custom setups.

**Comment by markknol** [comment]
Thanks for the additions

**Comment by nadako** [comment]
Thanks!

**Pull request merged**